src/test/java/org/rulez/demokracia/pdengine/UnimplementedTests.java

```diff
@@ -1,19 +1,32 @@
 package org.rulez.demokracia.pdengine;
+import static org.junit.Assert.assertTrue;
+
 import java.util.ArrayList;
+import java.util.List;
 
 import org.junit.Test;
 import org.rulez.demokracia.pdengine.annotations.TestedBehaviour;
 import org.rulez.demokracia.pdengine.annotations.TestedFeature;
 import org.rulez.demokracia.pdengine.annotations.TestedOperation;
-import org.rulez.demokracia.testhelpers.ThrowableTester;
+import org.rulez.demokracia.pdengine.testhelpers.CreatedDefaultChoice;
 
 @TestedFeature("Unimplemented")
 @TestedOperation("Unimplemented")
 @TestedBehaviour("Unimplemented")
-public class UnimplementedTests extends ThrowableTester {
-
+public class UnimplementedTests extends CreatedDefaultChoice {
```

Comment:
I cant find the original feature, operation, behaviour. :(

Reply:
Maybe there is no such thing.
When someone want to implement the feature, will find it here, and put the test into its right place.